code-challenges.test.js

```diff
@@ -13,25 +13,66 @@
 
 // Reminder: The test will call your function
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
+describe("when evenlyDivisible is called", () => {
+    it("returns '15 is divisible by three' when called with 15", () => {
+        //Arrange 
+        const num1 = 15;
+
+        //Act
+        const actualDivisible = evenlyDivisible(num1);
+
+        //Assert
+        expect(actualDivisible).toEqual("15 is divisible by three");
+    })
+    it("returns '0 is divisible by three' when called with 0", () => {
+        //Arrange 
+        const num2 = 0;
+    
+        //Act
+        const actualDivisible = evenlyDivisible(num2);
+    
+        //Assert
+        expect(actualDivisible).toEqual("0 is divisible by three");
+    })
+    it("returns '-7 is not divisible by three' when called with -7", () => {
+        //Arrange 
+        const num3 = -7;
+        
+        //Act
+        const actualDivisible = evenlyDivisible(num3);
+        
+        //Assert
+        expect(actualDivisible).toEqual("-7 is not divisible by three");
+    })
+})
 var num1 = 15
 // Expected output: "15 is divisible by three"
 
 var num2 = 0
 // Expected output: "0 is divisible by three"
 
 var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
 
+//create a function called evenlyDivisible
+// Takes in the argument of a number (num)
+const evenlyDivisible = (num) => {
+    // if the number is divisible by three return `${num} is divisible by three`
+    if (num%3 ===0){
+        return `${num} is divisible by three`
+    // if the number is not divisible by three return `${num} is not divisible by three`
+    }else if (num%3 !== 0){
```

Comment:
as was revealed when we went over this in class, because strings when used with modulo operator evaluate to a value that is not equal to 0, this `num%3 !== 0` actually evaluates to truthy and won't ever hit the else condition. Best practice is generally to ensure that the arguments that are provided are the correct data type and not proceed with any other logic if it's not.